src/genn/backends/cuda/backend.cc

```diff
@@ -534,6 +537,18 @@ void Backend::genDefinitionsPreambleInternal(CodeStream &os, const ModelSpecMerg
     if(getRuntimeVersion() >= 9000) {
         os <<"#include <cuda_fp16.h>" << std::endl;
     }
+    
+    // If NVTX profiling is enabled, include nvToolsExt header
+    if(getPreferences<Preferences>().enableNVTX) {
+        os << "#include <nvToolsExt.h>" << std::endl;
```

Comment:
As it's header only and thus easier to build, you should switch the NVTX3 which is included with:
```c++
#include "nvtx3/nvToolsExt.h"
```